cognee/api/v1/add/routers/get_add_router.py

```
@@ -2,7 +2,8 @@
 from fastapi import Form, UploadFile, Depends
 from fastapi.responses import JSONResponse
 from fastapi import APIRouter
-from typing import List, Optional
+from typing import List, Optional, Union, Dict
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)